e2e_samples/dataset_versioning/infra/terraform/.azdo/iac-template.yml

```diff
@@ -0,0 +1,71 @@
+parameters:
+- name: buildValidation
+  type: boolean
+  default: false
+- name: IaCRepo
+  type: string
+  default: infra
+- name: buildingEnvironment
+  type: string
+  default: staging
+jobs:
+- deployment: IaC
+  displayName: IaC
+  pool:
+    vmImage: 'Ubuntu-latest'
+  variables:
+  - group: terraform-secret-${{ parameters.buildingEnvironment }}
```

Comment:
you might need to add readme about how to create variable groups and enviroments in ADO? I think in the temperature-events folder there might be some reusable parts.

Reply:
Add some details on pipeline